static/sass/v1_section/_v1_cloud.scss

```diff
@@ -0,0 +1,104 @@
+// events on insights news feed, code ported from ../section/_cloud.scss
```

Comment:
I know this is snowflake code but can it be cleaned up to use BEM and Vanilla spacing variables?